mappers/bluetooth_mapper/README.md

```diff
@@ -0,0 +1,311 @@
+# Bluetooth Mapper
+
+
+## Introduction
+
+Mapper is an application that is used to connect  and control devices. This is an implementation of mapper for 
```

Comment:
nit: please remove the extra whitespace between `connect` & `and`

Reply:
done, thanks